src/arrowhead.jl

```diff
@@ -206,7 +206,7 @@ function materialize!(M::MatMulVecAdd{<:ArrowheadLayouts,<:AbstractStridedLayout
     for k = 1:d
         mul!(view(y, m+k:d:size(y,1)), A.D[k], view(x, n+k:d:size(x,1)), α, one(α))
     end
-    y_in
+    return y_in
```

Comment:
Why are you adding `return` everywhere? It's unnecessary.

Reply:
The CodeCov checks fail without the return. That's what the CodeCov was complaining about earlier, my tests had covered all the new code.

Reply:
Oh weird. I didn’t know adding return fixed the bug with codecov

annoying but whatevs